### ISSUE TITLE
Request: choosing what to interrupt or not

### ISSUE THREAD

**Issue by wolftune** [kmac/mindfulnotifier#53]
Although "Do Not Disturb" settings are respected, it would be nice to have activity-specific controls. So, most people would not want the mindful notifier to go off during a phone call. It might also be an option to not interrupt any audio that's playing, in that case only running if no other audio is happening.

**Comment by kmac** [comment]
Yeah, so the problem with this is that for me it's times like this when I often appreciate a mindfulness bell. You should maybe look into something like Tasker or Macrodroid to control your mute settings when Phone or your music app is enabled.

**Comment by wolftune** [comment]
The actual thing that prompted this was being on a phone call with the phone to my ear and then suddenly the bell went off at full speaker volume right in my ear and was awful. (Side-note: it was not something I experienced before because I only recently started having more calls where I was holding the phone to my ear, when I use a headset, it never bothered me to even interrupt a call)

I just added the other things imagining some flexibility people in general might want. I mostly don't want the bell to go off loudly in my ear when I have the phone to my head on a call, and secondarily, I probably don't want it to go off during phone calls. I'm personally fine with it interrupting music or something.

But I'll look into those other programs, neither is FLO though right? I saw neither in F-Droid. Either way, they are a workaround. It would be idea some day if MN at least somehow avoids the worst case that I experienced.

**Comment by mehmetgunacti** [comment]
I've been using MN for quite some time. Great app!

Except for one behavior: it rings the bell when playing Youtube videos or listening to music.

Is there a way of disabling MN (service) when certain apps are running? Through Tasker maybe?
